docs/dialect.md

```diff
@@ -60,7 +61,58 @@ return-type | Y | Indicates the return type of the function. For a list of allow
 
 - __date-function element__  
 The <span style="font-family: courier new">date-function</span> element is a specialized variant of <span style="font-family: courier new">function</span>. In addition to the base formula, you can specify one or more datepart formulas, which are used instead of the generic formula when available.  
-The function <span style="font-family: courier new">name</span> must be one of these: DATEADD, DATEDIFF, DATEFORMAT, DATENAME, DATEPARSE, DATEPART, DATETRUNC.  
+The function <span style="font-family: courier new">name</span> must be one of these: DATEADD, DATEDIFF, DATEFORMAT, DATENAME, DATEPARSE, DATEPART, DATETRUNC.   
```

Comment:
I'm not sure its even worth documenting DATEFORMAT, we don't implement it on most connectors internally.